.github/actions/build-llvm/action.yml

```diff
@@ -79,7 +87,9 @@ runs:
 
     - name: Install LLVM builder
       shell: ${{ runner.os == 'Windows' && 'msys2 {0}' || 'bash' }}
-      run: cargo install compiler-llvm-builder@${{ inputs.llvm-builder-version }}
+      run: |
+        cargo install compiler-llvm-builder --git https://github.com/matter-labs/era-compiler-llvm-builder --branch aba-rtti-support
+        # cargo install compiler-llvm-builder@${{ inputs.llvm-builder-version }}

```

Comment:
Change back to v1.0.35 after release of the new builder https://github.com/matter-labs/era-compiler-llvm-builder/pull/37